web/packages/plugins/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -447,16 +447,17 @@ function SelectDataSource(props: {
         marginTop: '10px',
       }}
     >
-      <MultiSelect
-        label="Select data sources to search"
-        initialSelectedItems={selectedDataSources}
-        handleSelectedItemsChange={(event) => {
+      <Autocomplete
```

Comment:
MultiSelect from EDS is depreciated. Substituted with the Autocomplete component as recomended by EDS
https://eds-storybook-react.azurewebsites.net/?path=/docs/inputs-select-multiselect--default